internal/util/time.go

```diff
@@ -6,5 +6,5 @@ import (
 )
 
 func Now(ctx context.Context) time.Time {
-	return time.Now()
+	return time.Now().UTC()
```

Comment:
Then also rename the function to NowUTC, otherwise we'll be confused in the future.